app/api/resources/user.py

```diff
@@ -276,10 +274,17 @@ def post(cls):
         if is_valid != {}:
             return is_valid, 400
 
-        result = DAO.create_user(data)
-
-        if result[1] is 200:
-            send_email_verification_message(data["name"], data["email"])
+        failed_send = ""
+        try:
+            send_email_verification_message(data['name'], data['email'])
+            
+        except:
```

Comment:
Please catch a specific exception. What is a potential exception you are expecting here? We should specifically handle that case. More on why it's better to catch specific exceptions rather than **Exception** here: https://stackoverflow.com/questions/4990718/about-catching-any-exception

Reply:
@ramitsawhney27, for instead of 500, I'm thinking perhaps 407 Proxy Authentication Required is more suitable in this error type coz it is related to authentication of the user's email. Will this be more appropriate?